subgraph/bin/introspect.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path')
+const fs = require('fs-extra')
+const { getIntrospectionQuery } = require('graphql')
+const { networks } = require('@unlock-protocol/networks')
+
+const introspectionJSONFilePath = path.join(
+  __dirname,
+  '..',
+  'introspection.json'
+)
+
+const endpoint = networks['5'].subgraph.endpointV2
+console.log(`Fetching graphql introspection from ${endpoint}`)
+async function main() {
```

Comment:
Can we just make this run every time but skip if CI=true?

Reply:
This way we can make sure never have an outdated version?
And also ideally instead of running agasing Goerli, we run against the "local" subgraph?

Reply:
hmmm i think we can add a flag to run against localhost but we should not default to localhost, as this will require running the whole stack locally to have a deployed subgraph and then query it for introspection.

What we can also add is a recurring ci task (such as the one for yarn lock or npm publish) that will run on prod branch and open a PR if the graph introspection file is outdated.

I think both of these are doable but can be addressed in another PR

Reply:
added a `yarn introspect:local` task that can be used locally to fetch against localhost